share.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Share
+excerpt: "Do you like ev3dev? Show your support! We invite you to star the repo on GitHub and submit your cool projects to our projects page."
```

Comment:
I guess I should have looked at the page first :blush: 

Still, I think the except should come from the Projects rather than the other stuff on the page.